composer.json

```diff
@@ -82,6 +81,9 @@
             "drupal/ctools": {
                 "Signature of EventDispatcherInterface::dispatch() has changed": "https://www.drupal.org/files/issues/2021-01-14/3191286-14.patch"
             },
+            "drupal/data_policy": {
+                "Compatibility with Drupal 9.3": "https://git.drupalcode.org/project/data_policy/-/merge_requests/15.diff"
```

Comment:
lets try to not use moving targets, if someone commits in your added MR it might break. Always create a local patch in the issue, or if you can't you can always open up an issue in Open Social and add a patch there for reference.

Reply:
This is some of the experiments done by me, as it is a draft PR. I will release a new version of module and change the version here.